src/server.js

```diff
@@ -73,6 +76,13 @@ module.exports = class Server {
       });
   }
 
+  optionsHandler(req, res) {
+    res.set('Access-Control-Allow-Origin', req.get('origin'));
+    res.set('Access-Control-Allow-Credentials', true);
+
+    res.sendStatus(204);
```

Comment:
I don't get it.. how is this different than what we're already doing inside of `getHandler`? Only the status right? Why is this necessary?